rust/.dockerignore

```diff
@@ -1,3 +1,6 @@
 target/
+!target/*/release/firezone-relay
```

Comment:
I think it's better to `mv` those before doing add in Dockerfile than excluding them from `.dockerignore`, just to make sure that we won't accidentally start adding binaries to containers that should not be there.

Reply:
good call